RestaurantMap/Scener/Home/HomeViewController.swift

```diff
@@ -94,17 +86,24 @@ class HomeViewController: UIViewController {
         restaurantMarkers.forEach({ $0.map = nil })
         restaurantMarkers.removeAll()
         for restaurant in restaurants {
-            let coordinate = CLLocationCoordinate2D(latitude: restaurant.geometry.location.lat, longitude: restaurant.geometry.location.lng)
+            let coordinate = CLLocationCoordinate2D(latitude: restaurant.coordinates.latitude, longitude: restaurant.coordinates.longitude)
             let marker = GMSMarker()
             marker.position = coordinate
-            marker.icon = UIImage(named: "current_marker_icon")
+            marker.icon = UIImage(named: "restaurant_marker_icon")
             marker.title = restaurant.name
-            marker.snippet = restaurant.plusCode.compoundCode
+            marker.snippet = restaurant.location.address
             marker.map = mapView
             self.restaurantMarkers.append(marker)
         }
     }
     
+    private func setupLocationDefault() {
+        updateCamera(locationCoordinate: GoogleMaps.defaultLocation.coordinate)
+        cleanMarker()
+        showMainMarker()
+        fetchRestaurantsNearbyLocation(locationCoordinate: GoogleMaps.defaultLocation.coordinate)
```

Comment:
remove this function from setup location function, cuz it's not related with the setupLocationDefault and maybe you can invoke it in next line where you invoke the setupLocationDefault()